### ISSUE TITLE
Retweet

### ISSUE THREAD

**Issue by yungfemo** [dante4rt/Galxe-AutoComplete-Tasks#2]
Hello 

Thank you so much for this automation. The issue when I use this code is that it completes the like task but doesnt do the retweet task please check again

**Comment by dante4rt** [comment]
> Hello
> 
> Thank you so much for this automation. The issue when I use this code is that it completes the like task but doesnt do the retweet task please check again

Yes, its not solve all tasks. But it can complete some of it, the rest need to be handled manually.
